developer/installation/reaction-cli.md

```diff
@@ -1,22 +1,43 @@
-### Reaction command-line-client (CLI)
+# Reaction command line cheatsheet
 
-Some handy commands to know
+Reaction ships with all the standard Meteor commands, as documented in the [Meteor Guides](https://docs.meteor.com/commandline.html), along with those from Reaction's own command line interface (CLI). Here are some handy commands to use while developing on Reaction:
 
-#### reaction --help
+## Reaction CLI
 
-`reaction -h` will give you help for the `reaction` command.
+### reaction -v
 
-#### reaction pull
+Before you get started using the Reaction CLI, it's important to make sure you're using the latest version. Use `reaction -v` to check what versions of Node, NPM, Meteor Node, Reaction CLI, Docker and what branch and version of Reaction you are currently running.
+
+```sh
+reaction -v
+
+Node: 9.5.0
+NPM: 5.6.0
+Meteor Node: 8.9.3
+Meteor NPM: 5.5.1
+Reaction CLI: 0.27.0
+Reaction: 1.8.0
+Reaction branch: release-1.8.0
+Docker: 17.12.0-ce
+```
+
+### reaction --help
+
+Shortcut: `-h`
+
+`reaction --help` or `-h` will give you help for the `reaction` command.
+
+### reaction pull
```

Comment:
I wonder if we shouldn't deprecate `reaction pull`. It's fairly problematic.

Reply:
Should we recommend `git fetch` instead?

Reply:
No.

I think you open up a whole new can of worms of trying to explain how git works. It all depends on how they have decided to set up their repo (I think it assumes that our repo is stil their `origin``). So I recommend just not trying to be prescriptive here and just leave it out.